src/scores.cpp

```diff
@@ -2288,7 +2379,7 @@ int loadGame(int player, int saveIndex)
 		}
 	}
 
-	if ( player == clientnum )
+	if ( players[c]->isLocalPlayer() )
```

Comment:
Where do we call loadGame() from now? just noticed this loadGame(0) in menu.cpp which may cause issues

https://github.com/TurningWheel/Barony/blob/bb26e9a0a087cedf69dd0f1c90239ab9dfe95c8b/src/menu.cpp#L10699

Reply:
What happens when you have a singleplayer save as clientnum == 2, and try to load a game? How does clientnum get set?

Reply:
clientnum is recorded in the save game and loaded back at scores.cpp:2081